### ISSUE TITLE
swift 2.3, swift 3 

### ISSUE THREAD

**Issue by asiliuk** [SwiftKitz/Appz#77]
Hello!
Do you have any plans on migrating library to newer versions of swift?

**Comment by Mazyod** [comment]
Hello!
For sure, please check out #76. Until Swift 3 is officially out, I can't make the main release use Swift 3. However, I can upgrade the library for Swift 2.3 if it's throwing warnings and errors?

**Comment by asiliuk** [comment]
Haven't tried to use it from project with 2.3 or 3
Maybe before official release you can create some swift3 branch and post updates there?

**Comment by Mazyod** [comment]
Makes sense, let's see where the PR contributor is on that front, and we can hopefully arrange something.

**Comment by Mazyod** [comment]
Swift 3 changes are in the `swift-3` branch for now, and will hopefully be merged into master as soon as Xcode 8 comes out.

**Comment by Mazyod** [comment]
Blocked by a Cocoapods bug now .. once I can overcome that bug, will release a new version.
